packages/wells/src/client/model/WellNPTFilter.ts

```diff
@@ -0,0 +1,25 @@
+import { ContainsAllOrAny } from './ContainsAllOrAny';
+import { LengthRange } from './LengthRange';
+
+export interface WellNPTFilter {
+  /**
+   * @type {LengthRange}
+   * @memberof WellNPTFilter
+   */
+  measuredDepth?: LengthRange;
+  /**
+   * @type {DoubleRange}
+   * @memberof WellNPTFilter
+   */
```

Comment:
Curious: What does these comments add? Both the type and the `memberof` property can be deduced from the code. Does it improve IDE integration? Doesn't typescript extract this info from the code already?

Reply:
I copied the pattern for the geospatial nodejs sdk. I haven't tested it properly yet and whether or not it works with or without, but it's mainly intended for the autogenerated docs. If you do `yarn docs` inside of the repo and open the `index.ts` in a live server, you'll get our autogenerated docs. 
![image](https://user-images.githubusercontent.com/25263842/118820060-36ed1d80-b8b6-11eb-827f-d59abe5c42b6.png)

If you go into any of the exported classes, you'll find a reference to memberof. 
![image](https://user-images.githubusercontent.com/25263842/118820407-87fd1180-b8b6-11eb-94a7-580f4ada26f5.png)

Once we have a stable service and a stable API we can deploy the autogenerated docs like the rest of Cognite.

Reply:
I'll create a card for releasing it.

Reply:
created [SDL-665]